.cspell.json

```diff
@@ -132,7 +132,8 @@
         "vscc",
         "wasm",
         "Xdai",
-        "goquorum"
+        "goquorum",
+        "outsh"
```

Comment:
@outsh I think you don't need this because image name in test code is updated to one that doesn't include this word.

Reply:
@petermetz There are 2 images, I'm still waiting for `ghcr.io/outsh/cactus-connector-corda-server` upload to hyperledger ghcr

Reply:
@outSH I see. thanks!

Reply:
> @izuru0 There are 2 images, I'm still waiting for `ghcr.io/outsh/cactus-connector-corda-server` upload to hyperledger ghcr

@outSH @izuru0 Apologies for the wait, I've just uploaded `ghcr.io/hyperledger/cactus-connector-corda-server:2022-04-18-a8a7ed1--1956`